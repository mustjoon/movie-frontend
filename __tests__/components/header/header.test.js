import { render } from "@testing-library/react";
import Header from "@/components/header/header";
import "@testing-library/jest-dom";

describe("Header", () => {
  it("renders ok", () => {
    const { container } = render(<Header />);
    expect(container).toMatchSnapshot();
  });
});
