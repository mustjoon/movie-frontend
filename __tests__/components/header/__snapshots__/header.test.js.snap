// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders ok 1`] = `
<div>
  <header
    class="bg-gray-900 text-white py-3"
  >
    <nav
      class="container mx-auto flex items-center justify-between"
    >
      <h1
        class="font-medium text-xl"
      >
        <a
          href="/"
        >
          Movie app
        </a>
      </h1>
      <ul
        class="flex items-center"
      >
        <li
          class="mr-6"
        >
          <a
            class="text-gray-400 hover:text-white cursor-pointer"
            href="/"
          >
            Home
          </a>
        </li>
        <li
          class="mr-6"
        >
          <a
            class="text-gray-400 hover:text-white"
            href="/movies/create"
          >
            Create
          </a>
        </li>
      </ul>
    </nav>
  </header>
</div>
`;
